pkg/cloud/ibmcloud/actuators/machine/actuator.go

```diff
@@ -129,6 +129,10 @@ func (ic *IbmCloudClient) Create(ctx context.Context, cluster *clusterv1.Cluster
 		return err
 	}
 
+	if guest == nil {
+		return fmt.Errorf("Guest host does not found")
+	}
```

Comment:
we are in guest create stage... so this info seems not very helpful 

you want to say that the create failed and we can't get the guest? @xunpan

Reply:
This is trying to keep logic correct.

`getGuest` will return `nil` with no error. I'm checking it and give a simple error message. If we do not make it, later `strconv.Itoa(*guest.Id)` will fail. Even we're creating machine at the time, I'm not sure if IBM Cloud or softlayer API failed to get `guest` in any cases. So, checking it for logic correct

Reply:
ok, I submitted a PR to add more info about the log, thanks for the detailed info~